### ISSUE TITLE
Evaluation Performance Mismatch

### ISSUE THREAD

**Issue by ZinYY** [pengzhangzhi/Open-dLLM#17]
Hi, thank you for sharing the repository. I ran the evaluation code using `bash run_eval.sh`, but I noticed that the performance results I obtained were lower than those reported in the README. Here are the results from my evaluation:

| Tasks          | Metric  |  Value |   | Stderr |
| -------------- | ------- | -----: | - | -----: |
| humaneval      | pass@1  | 0.1659 | ± | 0.0246 |
|                | pass@10 | 0.3354 | ± | 0.0370 |
| humaneval_plus | pass@1  | 0.1543 | ± | 0.0243 |
|                | pass@10 | 0.2805 | ± | 0.0352 |
| mbpp           | pass@1  | 0.1642 | ± | 0.0130 |
|                | pass@10 | 0.3640 | ± | 0.0215 |
| mbpp_plus      | pass@1  | 0.2332 | ± | 0.0168 |
|                | pass@10 | 0.4907 | ± | 0.0257 |

Do you have any suggestions on why these results might differ from those in the README? Thank you!

**Comment by pengzhangzhi** [comment]
ya thank you for raising this! we did some hyper-param searching in sampling, for example, temperature.  @zhangshuibai  can provides u those parameters for each task : )

**Comment by zhangshuibai** [comment]
Thanks for the issue. Can you try set the temperature to 0.6? This should give you better pass@1

**Comment by realgourmet** [comment]
Hi @zhangshuibai, thanks for the response but I still get roughly the same performance using temp=0.6. Here is my experiment result on humaneval:
```
      "pass@1,create_test": 0.1774390243902439,
      "pass@1_stderr,create_test": 0.025924853412781067,
      "pass@10,create_test": 0.3231707317073171,
      "pass@10_stderr,create_test": 0.036632096446028474
```

I understand the results from README are from your paper [[2509.23405] Planner Aware Path Learning in Diffusion Language Models Training](https://www.arxiv.org/abs/2509.23405). Would it be possible to provide the exact eval hyperparameters to reproduce the results there? Thank you!
